README.md

```diff
@@ -119,8 +118,80 @@ images:
     newTag: dev
 ```
 
+Finally apply these changes:
+
+    $ kubectl apply -k ./k8s/operator/
+
 See the [operator/README.md](https://github.com/3box/keramik/blob/main/operator/README.md) for details on certain design patterns of the operator.
 
+## IPFS
+
+The IPFS behavior used by Ceramic can be customized.
+
+### Rust IPFS
+
+Example network config that uses Rust based IPFS (i.e. ceramic-one) with its defaults.
+
+```yaml
+apiVersion: "keramik.3box.io/v1alpha1"
+kind: Network
+metadata:
+  name: example-vanilla-kubo
+spec:
+  replicas: 5
+  ceramic:
+    ipfs:
+      kind: rust
```

Comment:
defaults are nice 

and this looks like all nodes with same version, eventually mixed networks might be useful